iidm/iidm-converter-api/src/main/java/com/powsybl/iidm/tools/ConversionToolUtils.java

```diff
@@ -100,6 +102,23 @@ public static Properties readProperties(CommandLine line, OptionType optionType,
         return properties;
     }
 
+    public static Option createSkipPostProcOption() {
+        return Option.builder().longOpt(SKIP_POSTPROC)
+                .desc("skip network importer post processors (when configured)")
+                .build();
+    }
+
+    public static ImportConfig createImportConfig(CommandLine line) {
+        return createImportConfig(line, ImportConfig.load());
```

Comment:
I wonder if this kind of "hidden" read of the platform config may cause issues, in particular in tools unit tests.
We can postpone this question until we actually meet a problem (or not).

Reply:
I deleted this method as it may indeed cause some confusions. `ImportConfig.load()` is "hidden" in `DefaultConversionOption` but I think it might cause less issues since this object must in any case be overriden in tests in most cases.